requirements.txt

```diff
@@ -0,0 +1,28 @@
+aniso8601==9.0.1
```

Comment:
in the requirements file u should put just the requirements that u installed, pip freeze shows u all the packages that are installed, it includes what u installed and their dependencies